jbpm-workitems/jbpm-workitems-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -66,58 +67,25 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
+      <scope>provided</scope>
     </dependency>
 
     <dependency>
-      <groupId>${project.groupId}</groupId>
+      <groupId>org.jbpm.contrib</groupId>
       <artifactId>template-resources</artifactId>
       <scope>provided</scope>
+      <version>${archetypeVersion}</version>
```

Comment:
can we use ${version.org.kie} here instead? Tobe in sync with the plugin config.

Reply:
I don’t think it’s necessary, the version needs algin to archetype's version not kie's version.

Reply:
> can we use ${version.org.kie} here instead? Tobe in sync with the plugin config.
I don’t think it’s necessary, the version needs algin to archetype's version not kie's version.

Reply:
Look in the plugins section below, there you have `<resourceBundle>org.jbpm.contrib:template-resources:${version.org.kie}</resourceBundle>` so we're actually depending on the `${version.org.kie}` defined in parent. So align this one way or the other.

Reply:
done